hw/top_earlgrey/rdc/chip_earlgrey_asic_scenario.tcl

```diff
@@ -4,8 +4,63 @@
 #
 # Meridian RDC Reset Scenario
 
+###############
+# Reset Group #
+###############
+
+# List of external reset group
+set ext_rst_lst {}
+# POR_N active low
+lappend ext_rst_lst [list { POR_N } {reset low}]
+
+foreach_in_collection j [get_pins top_earlgrey.rstmgr_aon_resets.*] {
+    lappend ext_rst_lst [list [get_attribute ${j} full_name] {reset low}]
+}
+
+set_reset_group $ext_rst_lst -name ext_rst
+
 ###################
 # Reset Scenarios #
 ###################
 
 # TODO: SPI_CSB asserted(release from reset) only when SYS_RST_N is not asserted
+
+
+# POR_N
+set_reset_scenario { {POR_N {reset { @t0 0 } { #10 1}}}} -name ScnPOR
+
+# AST POK
+
```

Comment:
do we want me to point out what these are?

Reply:
Yep. Would be great to create a simple scenario (assertion -> deassertion) for POK signals generating from AST.